scripts/svgGraphs.js

```diff
@@ -70,6 +70,68 @@ const {barLineGraph, stackedAreaGraph, pieGraph} = (function () {
         }
     }
 
+    const nodeIterator = (elem, fn) => {
+        fn(elem);
+        for (const child of elem.children) {
+            nodeIterator(child, fn);
+        }
+    };
+
+    const createSvgText = (svgElement) => {
+        const classNames = new Set(['graph-background']);
+
+        nodeIterator(svgElement, (elem) => {
+            elem.classList.forEach(className => {
+                classNames.add(className);
+            });
+        });
+
+        const dummyElement = document.createElement('a');
+        classNames.forEach(className => {
+            dummyElement.classList.add(className);
+        });
+        const cssStyleRules = new Set();
+        for (const sheet of document.styleSheets) {
+            for (const rule of sheet.cssRules) {
+                if (dummyElement.matches(rule.selectorText)) {
+                    cssStyleRules.add(rule.cssText);
+                }
+            }
+        }
+
+        const width = window.getComputedStyle(svgElement).width;
+        let svgText = svgElement.outerHTML.replace('<svg ', `<svg xmlns="http://www.w3.org/2000/svg" width="${width}" `);
+        const endOfFirstElement = svgText.indexOf('>');
+        svgText = `${svgText.slice(0, endOfFirstElement + 1)}
+            <style>
+                ${[...cssStyleRules].join('')}
+            </style>
+            <rect width="100%" height="100%" class="graph-background" />
+            ${svgText.slice(endOfFirstElement + 1)}`;
+        return svgText;
+    };
+
+    function addDownloadButton(svg, config) {
+        if (config.downloadLabel) {
+            const svgElement = svg.nodes()[0];
+            const downloadButton = document.createElement('button');
+            downloadButton.textContent = config.downloadLabel;
+            downloadButton.classList.add('grpah-download-button');
```

Comment:
```suggestion
            downloadButton.classList.add('graph-download-button');
```
(and in `layout.less` -- at least they matched! :D)

Reply:
I pushed a commit to your branch to fix this.

Reply:
Thanks, I forgot to come back to that!